bin/nest-server

```diff
@@ -1,33 +1,27 @@
 #!/bin/bash
 
-BUFFER_SIZE="${NEST_SERVER_BUFFER_SIZE:-4096}"
-DAEMONIZE="${NEST_SERVER_DAEMONIZE:-0}"
+DAEMON="${NEST_SERVER_DAEMON:-0}"
 HOST="${NEST_SERVER_HOST:-127.0.0.1}"
-PLUGIN="${NEST_SERVER_PLUGIN:-}"
 PORT="${NEST_SERVER_PORT:-5000}"
 STDOUT="${NEST_SERVER_STDOUT:-0}"
-USER="${NEST_SERVER_USER:-${USER:-65534}}"
 
 usage() {
   echo "NEST Server"
   echo "-----------"
-  echo "Usage: nest-server log|status|start|stop|restart [-d] [-o] [-h <HOST>] [-p <PORT>] [-P <PLUGIN>] [-u <UID>]"
+  echo "Usage: nest-server log|status|start|stop|restart [-D] [-h <HOST>] [-o] [-p <PORT>]"
   echo ""
   echo "Commands:"
-  echo "  log         display the sever log stored in /tmp/nest-server.log"
-  echo "  status      display the status of NEST Server"
-  echo "  start       start a new instance of the server"
-  echo "  stop        stop a server instance running on <HOST>:<PORT>"
-  echo "  restart     restart (i.e. stop and start) a server on <HOST>:<PORT>"
+  echo "  log         display the server log stored in /tmp/nest-server.log"
+  echo "  status      display the status of all server instances"
+  echo "  start       start a server instance on <HOST>:<PORT>"
+  echo "  stop        stop a server instance on <HOST>:<PORT>"
+  echo "  restart     restart (i.e. stop and start) a server instance on <HOST>:<PORT>"
   echo
   echo "Options:"
-  echo "  -b <BUFFER_SIZE>   set internal buffer size [default: 4096]"
-  echo "  -d                 daemonize the server process"
+  echo "  -D                 daemonize the server process"
```

Comment:
Is this change motivated by some default for other software? I'm not sure I like this if it is not.